test/lib/index.ts

```diff
@@ -22,6 +22,31 @@ test('Parse npm package-lock.json', async (t) => {
   t.deepEqual(depTree, JSON.parse(expectedDepTree), 'Tree generated as expected');
 });
 
+test('Parse npm package-lock.json with devDependencies', async (t) => {
```

Comment:
`hasDevDependencies` can we assert here it is true?

Reply:
@lili2311 It's in a fixture - https://github.com/snyk/nodejs-lockfile-parser/pull/15/files/355f3fb209d6e24d9b36929c328db87e5a7203cf#diff-00d698a5881a0baea201915c8519647cR4